examples/common/python/setup.py

```diff
@@ -55,6 +60,7 @@
     compile_args += ['-g']
 
 include_dirs = [
+    os.path.join(tcf_root_dir, 'tc/sgx/common/verify_ias_report'),
```

Comment:
split include_dirs for verify_ias module. Crypto module doesn''t need verify_ias_report

Reply:
Done